File: src/main/kotlin/de/ithock/fishlang/FishLanguage.java
package de.ithock.fishlang;

import com.intellij.lang.Language;

/*
d93f9
ff5555
ff79c6
 */
public class FishLanguage extends Language {

    public static final FishLanguage INSTANCE = new FishLanguage();

    private FishLanguage() {
        super("Fish");
    }

}

File: src/main/kotlin/de/ithock/idea/fishlang/FishLanguage.kt
/*
 * Copyright (c) 2021 - 2022 IT-Hock. All rights reserved
 */
package de.ithock.idea.fishlang

import com.intellij.lang.Language

class FishLanguage : Language("Fish") {
    companion object {
        val INSTANCE: FishLanguage = FishLanguage()
    }
}